docs/runbooks/rob-381-upbit-index-altseason-recon.md

```
@@ -0,0 +1,159 @@
+
+**Status:** PR1 reconnaissance only. No production MCP tool / collector built here.
+**Parent:** ROB-377 (코인 마켓레짐·파생심리 데이터 소스 — A1/A2 완료) · ROB-369 (crypto 리포트 데이터 공백)
+**Verdict:** `implement` — **분할 권장** (아래 [Final verdict](#final-verdict) 참고).
+**Date:** 2026-05-31
+
+---
+
+## Why this issue exists (context)
+
+ROB-377은 #1040 (CoinGecko `/global` 기반 crypto market index / dominance / total market cap)와
+#1045 (Binance USD-M OI + LSR)로 마켓레짐·파생심리 수용 기준을 충족했다. 그 본문에
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)